src/containers/menu/TrackMenu.tsx

```diff
@@ -249,11 +223,7 @@ const TrackMenu = (props: TrackMenuProps) => {
 
   const menu = getMenu()
 
-  return (
-    <CascadingMenu menu={menu} mount={props.mount} className={props.className}>
-      {props.children}
-    </CascadingMenu>
-  )
+  return props.children(menu.items)
```

Comment:
I like the FaCC pattern choice here, but we should chat about how we feel as a team. afaik, this may be the first use?

Reply:
There isn't much reason to have these as components, my plan is to change them into functions that accept `dispatch` in my next PR. I can't decide where in the directory structure functions like that should live though. `containers` doesn't really make sense, but neither does `utils` or `selectors`. Any thoughts?

Reply:
maybe a `helpers` file or something like that near the component that uses it w/ a sub-folder containing both?

i think we should probably get rid of the containers vs. components structure over time, esp. with useDispatch, etc.

Reply:
Ok I like that. And I agree, the container/component distinction doesn't provide much value when using the redux hooks